src/example.ts

```diff
@@ -112,8 +114,8 @@ const serviceProviderConfig: IServiceProviderConfig = {
   ],
 };
 
-const redisClient = redis.createClient({
-  host: "redis",
+const redisClient: redis.RedisClientType = redis.createClient({
```

Comment:
I'm afraid of this manual casting. Are we sure `redis.createClient` returns a `redis.RedisClientType`? 
Check the following snippet in your IDE:
```ts
type Equals<X, Y> = X extends Y
  ? Y extends X
    ? "they are the same type"
    : "Y is narrower than X"
  : Y extends X
  ? "X is narrower than Y"
  : "they are different";

type IS_EQUEAL = Equals<
  redis.RedisClientType,
  ReturnType<typeof redis.createClient>
>;
```

The type of `IS_EQUEAL` is `"Y is narrower than X"`, meaning that `redis.RedisClientType` has more attributes than `ReturnType<typeof redis.createClient>`. Is it correct? 

([playground](https://www.typescriptlang.org/play?#code/C4TwDgpgBAKgjFAvFA3gMwPYYFxWAJwFcIAaKAIwEN9cBnAgSwDsBzAXwChRJYAmJVJhx4iETt2gwAzAJRUAXriaEAtuQj5OXcNACiAR0KUANrQA8ADTIBNAHwCLUCAA9gEJgBNaUaxyhQAfh8nV3cvKAs-f0CoACJgAAsIEChqaEToWkoVdJ1YqP9cWOsoBm8manwMAHcNPATKJgj8wuCXN09vSP8g2McyqAr8Ktr8esafFqgijJS0qA8GNDQNd2BYgG4ObR5dAEUAfTh+ZAMjUzN4MhheWy2JKH2jmVPDE3Mr2Ck7nb1D3hejzeFxu12+9x0j0OcAQr3OHzg1zgPyAA))

Reply:
Fixed in [2fd4860](https://github.com/pagopa/io-spid-commons/pull/133/commits/2fd4860c4d8556be0a83c6d5136dceec6546c235)